### PR TITLE
Add new Route to display a poll after creation and fetch the poll with the info from url

### PR DESCRIPTION
**Pull request by matt989253**
Linked to issue #21

### CODE REVIEW

File: src/Api/Polls.elm [hunk 1]
```diff
@@ -38,6 +45,10 @@ type alias Poll =
     }
 
 
+type alias PollIdentifier =
```
Comment (alexandrepiveteau) [inline src/Api/Polls.elm]: `PollDiscriminator` is maybe a better name, if we want to have indicate it's the discriminating field of the weak entity.

File: src/Main.elm [hunk 2]
```diff
@@ -178,6 +186,7 @@ type Message
     | AuthMessage Auth.Message
     | PollMessage Poll.Message
     | NewPollMessage NewPoll.Message
+    | DisplayPollMessage NewPoll.Message
```
Comment (alexandrepiveteau) [inline src/Main.elm]: Why do we want two types of messages here ? We could do it like Auth, no ?

File: src/Main.elm [hunk 3]
```diff
@@ -34,6 +34,7 @@ type PageModel
     | QuitModel Quit.Model
     | PollModel Poll.Model
     | NewPollModel NewPoll.Model
+    | DisplayPollModel NewPoll.Model
```
Comment (alexandrepiveteau) [inline src/Main.elm]: Same model, no ?

File: src/Page/DisplayPoll.elm [hunk 4]
```diff
@@ -140,23 +169,33 @@ view model =
...
 
 
+extractTitle : Model -> String
+extractTitle model =
+    case model.poll of
+        Just poll ->
+            poll.title
+
+        Nothing ->
+            ""
```
Comment (alexandrepiveteau) [inline src/Page/DisplayPoll.elm]: Use `Maybe.withDefault`.

**Review by alexandrepiveteau — CHANGES_REQUESTED** [review]

File: src/Main.elm [hunk 5]
```diff
@@ -351,7 +334,7 @@ changeRouteTo route model =
                     initWith
                         NewPollMessage
```
Comment (alexandrepiveteau) [inline src/Main.elm]: Is this expected ?

**Review by alexandrepiveteau — APPROVED** [review]

File: src/Main.elm (hunk 5 above)
Comment (alexandrepiveteau) [reply src/Main.elm]: (my bad, the code was not up-to-date)

Comment (matt989253) [reply src/Main.elm]: This was fixed in 8c4c5e4

**Pull request merged**